ctx.go

```diff
@@ -244,47 +244,6 @@ func (c *Ctx) UsePrivateKey(key PrivateKey) error {
 	return nil
 }
 
-// UserCertAndNullKey configures the context to use the given certificate
-// and NULL private key for the SSL handshakes.
-// It allows you to use private keys that are never accessible directly
-// e.g.: to which openssl has access only via Engine module.
-// https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_use_cert_and_key.html
-func (c *Ctx) UseCertAndExternalKey(cert *Certificate) error {
```

Comment:
@merlin-northern do you have a plan for these? If so, there needs to be an alternative definition in the shim for openssl 1.0.

Reply:
for now they are not used, since client in current implementation will use [UseCertificate](https://github.com/mendersoftware/mender/pull/588/commits/bc9a60bd281b77cf148e3912f8f6428503004bd5#diff-1cd41a2611da49ce63630d5a51bd82c5R315) and [UsePrivateKey](https://github.com/mendersoftware/mender/pull/588/commits/bc9a60bd281b77cf148e3912f8f6428503004bd5#diff-1cd41a2611da49ce63630d5a51bd82c5R341) so this can be safely removed now.